src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingOverrideCheck.java

```diff
@@ -216,21 +218,20 @@ public void setJavaFiveCompatibility(final boolean compatibility) {
         return getRequiredTokens();
     }
 
+    @Override
+    public boolean isCommentNodesRequired() {
+        return true;
+    }
+
     @Override
     public int[] getRequiredTokens() {
         return new int[]
         {TokenTypes.METHOD_DEF, };
     }
 
-    // -@cs[CyclomaticComplexity] Too complex to break apart.
-    // suppress deprecation until https://github.com/checkstyle/checkstyle/issues/11166
-    @SuppressWarnings("deprecation")
     @Override
     public void visitToken(final DetailAST ast) {
-        final TextBlock javadoc =
-            getFileContents().getJavadocBefore(ast.getLineNo());
```

Comment:
Where is the removal suppression?
https://github.com/haanhvu/checkstyle/commit/14391708cba2675bb2cd56135533bc82d6c120c2#diff-c1ffc75b7152435a4d5d40d7ee9b32b57402959b1ad471a41e753ea67554378b

Reply:
fixed